### PR TITLE
Move equinix-ocp-metal jobs to new ofcir server

### PR DESCRIPTION
**Pull request by derekhiggins**

### CODE REVIEW

**Comment by derekhiggins** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.11-ocp-e2e-metal-ipi-ovn-arm64 periodic-ci-openshift-release-master-nightly-4.11-e2e-metal-ipi-bm

**Comment by derekhiggins** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.11-ocp-e2e-metal-ipi-ovn-arm64 periodic-ci-openshift-release-master-nightly-4.11-e2e-metal-ipi-bm

**Comment by derekhiggins** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.11-ocp-e2e-metal-ipi-ovn-arm64 periodic-ci-openshift-release-master-nightly-4.12-e2e-metal-ipi-sdn-bm periodic-ci-openshift-release-master-nightly-4.11-e2e-metal-ipi-bm periodic-ci-openshift-release-master-nightly-4.14-e2e-metal-ipi-ovn-ipv6

**Comment by derekhiggins** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.11-ocp-e2e-metal-ipi-ovn-arm64 periodic-ci-openshift-release-master-nightly-4.12-e2e-metal-ipi-sdn-bm periodic-ci-openshift-release-master-nightly-4.11-e2e-metal-ipi-bm periodic-ci-openshift-release-master-nightly-4.14-e2e-metal-ipi-ovn-ipv6

**Comment by derekhiggins** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.11-ocp-e2e-metal-ipi-ovn-arm64 periodic-ci-openshift-release-master-nightly-4.12-e2e-metal-ipi-sdn-bm periodic-ci-openshift-release-master-nightly-4.11-e2e-metal-ipi-bm periodic-ci-openshift-release-master-nightly-4.14-e2e-metal-ipi-ovn-ipv6 pull-ci-openshift-metal3-dev-scripts-master-e2e-agent-compact periodic-ci-openshift-release-master-nightly-4.10-e2e-metal-ipi-ovn-ipv6

**Comment by derekhiggins** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.14-e2e-metal-ipi-ovn-ipv6 periodic-ci-openshift-release-master-nightly-4.10-e2e-metal-ipi-ovn-ipv6 periodic-ci-openshift-release-master-nightly-4.13-e2e-metal-ipi-sdn-serial-virtualmedia-bond

**Comment by derekhiggins** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.11-ocp-e2e-metal-ipi-ovn-arm64 periodic-ci-openshift-release-master-nightly-4.12-e2e-metal-ipi-sdn-bm periodic-ci-openshift-release-master-nightly-4.11-e2e-metal-ipi-bm periodic-ci-openshift-release-master-nightly-4.14-e2e-metal-ipi-ovn-ipv6 pull-ci-openshift-metal3-dev-scripts-master-e2e-agent-compact periodic-ci-openshift-release-master-nightly-4.10-e2e-metal-ipi-ovn-ipv6

**Comment by derekhiggins** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.14-e2e-metal-ipi-ovn-ipv6 periodic-ci-openshift-release-master-nightly-4.10-e2e-metal-ipi-ovn-ipv6 periodic-ci-openshift-release-master-nightly-4.13-e2e-metal-ipi-sdn-serial-virtualmedia-bond

**Comment by derekhiggins** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.12-e2e-metal-ipi-sdn-bm periodic-ci-openshift-release-master-nightly-4.11-e2e-metal-ipi-bm

**Comment by derekhiggins** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.14-e2e-metal-ipi-ovn-ipv6 periodic-ci-openshift-release-master-nightly-4.10-e2e-metal-ipi-ovn-ipv6 periodic-ci-openshift-release-master-nightly-4.12-e2e-metal-ipi-sdn-serial-ipv4 periodic-ci-openshift-release-master-nightly-4.8-e2e-metal-ipi-virtualmedia periodic-ci-openshift-release-master-nightly-4.11-e2e-metal-ipi-ovn-ipv4

**Comment by derekhiggins** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.14-e2e-metal-ipi-ovn-ipv6 periodic-ci-openshift-release-master-nightly-4.10-e2e-metal-ipi-ovn-ipv6 periodic-ci-openshift-release-master-nightly-4.12-e2e-metal-ipi-sdn-serial-ipv4 periodic-ci-openshift-release-master-nightly-4.8-e2e-metal-ipi-virtualmedia periodic-ci-openshift-release-master-nightly-4.11-e2e-metal-ipi-ovn-ipv4

**Comment by derekhiggins** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.11-ocp-e2e-metal-ipi-ovn-arm64 periodic-ci-openshift-release-master-nightly-4.10-e2e-metal-ipi-ovn-ipv6 periodic-ci-openshift-release-master-nightly-4.11-e2e-metal-ipi-bm periodic-ci-openshift-release-master-nightly-4.11-e2e-metal-ipi-ovn-ipv4 periodic-ci-openshift-release-master-nightly-4.12-e2e-metal-ipi-sdn-bm periodic-ci-openshift-release-master-nightly-4.12-e2e-metal-ipi-sdn-serial-ipv4 periodic-ci-openshift-release-master-nightly-4.13-e2e-metal-ipi-sdn-serial-virtualmedia-bond periodic-ci-openshift-release-master-nightly-4.14-e2e-metal-ipi-ovn-ipv6 periodic-ci-openshift-release-master-nightly-4.8-e2e-metal-ipi-virtualmedia pull-ci-openshift-metal3-dev-scripts-master-e2e-agent-compact

**Comment by derekhiggins** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.11-ocp-e2e-metal-ipi-ovn-arm64 periodic-ci-openshift-release-master-nightly-4.10-e2e-metal-ipi-ovn-ipv6 periodic-ci-openshift-release-master-nightly-4.11-e2e-metal-ipi-bm periodic-ci-openshift-release-master-nightly-4.11-e2e-metal-ipi-ovn-ipv4 periodic-ci-openshift-release-master-nightly-4.12-e2e-metal-ipi-sdn-bm periodic-ci-openshift-release-master-nightly-4.12-e2e-metal-ipi-sdn-serial-ipv4 periodic-ci-openshift-release-master-nightly-4.13-e2e-metal-ipi-sdn-serial-virtualmedia-bond periodic-ci-openshift-release-master-nightly-4.14-e2e-metal-ipi-ovn-ipv6 periodic-ci-openshift-release-master-nightly-4.8-e2e-metal-ipi-virtualmedia pull-ci-openshift-metal3-dev-scripts-master-e2e-agent-compact

**Comment by derekhiggins** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.11-e2e-metal-ipi-bm periodic-ci-openshift-release-master-nightly-4.12-e2e-metal-ipi-sdn-bm

**Comment by derekhiggins** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.11-ocp-e2e-metal-ipi-ovn-arm64 periodic-ci-openshift-release-master-nightly-4.10-e2e-metal-ipi-ovn-ipv6 periodic-ci-openshift-release-master-nightly-4.11-e2e-metal-ipi-bm periodic-ci-openshift-release-master-nightly-4.11-e2e-metal-ipi-ovn-ipv4 periodic-ci-openshift-release-master-nightly-4.12-e2e-metal-ipi-sdn-bm periodic-ci-openshift-release-master-nightly-4.12-e2e-metal-ipi-sdn-serial-ipv4 periodic-ci-openshift-release-master-nightly-4.13-e2e-metal-ipi-sdn-serial-virtualmedia-bond periodic-ci-openshift-release-master-nightly-4.14-e2e-metal-ipi-ovn-ipv6 periodic-ci-openshift-release-master-nightly-4.8-e2e-metal-ipi-virtualmedia pull-ci-openshift-metal3-dev-scripts-master-e2e-agent-compact

**Comment by derekhiggins** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.11-e2e-metal-ipi-bm periodic-ci-openshift-release-master-nightly-4.11-e2e-metal-ipi-ovn-ipv4 periodic-ci-openshift-release-master-nightly-4.12-e2e-metal-ipi-sdn-bm periodic-ci-openshift-release-master-nightly-4.14-e2e-metal-ipi-ovn-ipv6

**Comment by derekhiggins** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.11-e2e-metal-ipi-bm

**Comment by derekhiggins** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.14-e2e-agent-sno-ipv6

**Comment by derekhiggins** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.11-e2e-metal-ipi-bm

**Comment by derekhiggins** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.11-e2e-metal-ipi-bm

**Comment by derekhiggins** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.11-e2e-metal-ipi-bm

File: ci-operator/step-registry/ofcir/acquire/ofcir-acquire-commands.sh [hunk 1]
```diff
@@ -100,7 +100,7 @@ function getCIR(){
 #CLUSTERTYPE can be one of "virt", "virt-arm64" or "baremetal"
 CIRTYPE=host
 [ "$CLUSTERTYPE" == "baremetal" ] && CIRTYPE=cluster
-[ "$CLUSTERTYPE" == "virt-arm64" ] && CIRTYPE=host-arm
+[ "$CLUSTERTYPE" == "virt-arm64" ] && CIRTYPE=host_arm
```
Comment (andfasano) [inline ci-operator/step-registry/ofcir/acquire/ofcir-acquire-commands.sh:103]: @derekhiggins did we maybe documented this one already?

Comment (derekhiggins) [reply ci-operator/step-registry/ofcir/acquire/ofcir-acquire-commands.sh:103]: I've updated the maintanence doc to include all the correct pools we use

**Comment by derekhiggins** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.11-e2e-metal-ipi-bm

**Comment by derekhiggins** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.11-e2e-metal-ipi-bm

**Comment by derekhiggins** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.11-e2e-metal-ipi-bm
going to try and access this one while it runs

**Comment by derekhiggins** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.11-e2e-metal-ipi-bm
BM node missing nic, removed from the pool

**Comment by derekhiggins** [comment]
/pj-rehearse ack
the tests in periodic-ci-openshift-release-master-nightly-4.11-e2e-metal-ipi-bm  passed 
the job failed because it failed to release the node on ofcir, this happened because I removed the node from ofcir for debugging purposes

**Comment by derekhiggins** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.12-e2e-metal-ipi-sdn-bm periodic-ci-openshift-release-master-nightly-4.14-e2e-metal-ipi-ovn-ipv6

**Comment by derekhiggins** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.14-e2e-metal-ipi-sdn-bm periodic-ci-openshift-release-master-nightly-4.13-e2e-metal-ipi-sdn-bm

**Comment by deepsm007** [comment]
Can we execute 1 arm periodic as well to verify it works on arm?

**Comment by aleskandro** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.12-arm64-stable-baremetalds-ipi-ovn-ipv4-f14

**Comment by deads2k** [comment]
approving to fix CI signal for our metal jobs.

**Pull request merged**